src/System.Runtime.Serialization.Primitives/src/Configurations.props

```diff
@@ -4,7 +4,7 @@
     <BuildConfigurations>
       netcore50aot-Windows_NT;
       uap101aot-Windows_NT;
-      net463-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
why we are specifying the OS configuration here and not in other configurations files which has netfx?